Project.toml

```
@@ -44,6 +45,7 @@ TuringDynamicHMCExt = "DynamicHMC"
 TuringOptimExt = "Optim"
 
 [compat]
+ADTypes = "0.1.5"
```

ADTypes 0.1.5 is quite old, we should support at least
```suggestion
ADTypes = "0.2"
```